package.json

```diff
@@ -43,43 +43,40 @@
     "@angular/platform-browser": "7.2.16",
     "@angular/platform-browser-dynamic": "7.2.16",
     "@angular/router": "7.2.16",
-    "@blackbaud/auth-client": "2.21.0",
+    "@blackbaud/auth-client": "2.24.0",
     "@blackbaud/skyux-lib-clipboard": "1.1.1",
-    "@blackbaud/skyux-lib-code-block": "1.3.1",
+    "@blackbaud/skyux-lib-code-block": "1.4.0",
     "@blackbaud/skyux-lib-media": "1.2.0",
-    "@blackbaud/skyux-lib-stache": "3.2.3",
-    "@pact-foundation/pact": "9.6.0",
-    "@pact-foundation/pact-web": "9.6.0",
-    "@skyux-sdk/builder": "3.13.0",
-    "@skyux-sdk/builder-plugin-skyux": "1.2.1",
+    "@blackbaud/skyux-lib-stache": "3.3.0",
+    "@skyux-sdk/builder": "3.16.2",
+    "@skyux-sdk/builder-plugin-skyux": "1.3.1",
     "@skyux-sdk/e2e": "3.1.2",
-    "@skyux-sdk/pact": "3.3.1",
```

Comment:
Yeah, the pact stuff isn't needed for our components, and it will be removed in the next version of SKY UX, so I thought we could go ahead and remove it now. It shouldn't have any effect on our consumers or the build result.